api/src/modules/characters/controllers/character.ctrl.ts

```diff
@@ -0,0 +1,29 @@
+import { AppRequestHandler } from 'express';
+import boom from '@hapi/boom';
+import status from 'http-status-codes';
+
+import { logger } from '../../../shared/logger';
+import { createNewCharacterForUser } from '../characters.context';
+
+interface CreateNewCharacterReq {
+  body: {
+    name: string;
+    race: string;
+  };
+}
+export const createNewCharacter: AppRequestHandler<CreateNewCharacterReq> = async (
+  req,
+  res
+) => {
+  const { name, race } = req.body;
+  const user = req.user;
+
+  try {
+    const character = await createNewCharacterForUser(user, name, race);
+    logger.info(`Character "${name} with race ${character.race}" created`);
+
+    return res.status(status.OK).send({ id: character.id });
```

Comment:
it's good practice to return entire created resource, I learn it some time ago.
it's because it's often good on frontend to show the new created resource instantly and later just updated it with responses. in most case it will be the same u send, but sometimes it can be subtely different, e.g. if we would lowercase all chars names for some reason (we won't, it just an example)